desktop/plugins/hermesdebuggerrn/index.tsx

```diff
@@ -21,9 +21,12 @@ import Banner, {isBannerEnabled} from './Banner';
 import SelectScreen from './SelectScreen';
 import ErrorScreen from './ErrorScreen';
 import ChromeDevTools from './ChromeDevTools';
+import {parseEnvironmentVariableAsNumber} from '../../app/src/utils/environmentVariables';
 
 const POLL_SECS = 5 * 1000;
-const METRO_HOST = 'http://localhost:8081';
+const METRO_HOST = 'localhost';
+const METRO_PORT = parseEnvironmentVariableAsNumber('METRO_SERVER_PORT', 8081);
+const METRO_URL = `http://${METRO_HOST}:${METRO_PORT}`;
```

Comment:
Just to be on the safe side, could you replace this with the following just to make sure no unexpected injections are possible?

```js
const METRO_URL = new URL('http://localhost');
METRO_URL.port = METRO_PORT;
```

and then use `METRO_URL.toString()` to get the url below.

Reply:
6fd4b56f3069e5d93f6f145425509cb18918ac5d
I had to cancel importing `parseEnvironmentVariableAsNumber` as I didn't notice it was in different packages.
Using it as a string anyway for `URL` ctor so parsing routine is a bit different.

Reply:
FYI I just figured why I couldn't built locally : I was on node 10 or 12, and metro couldn't build, finally worked it out by switching to node 15. The doc seems outdated on this one (node >= 8).